Content.Server/Lightning/LightningSystem.cs

```diff
@@ -48,15 +49,20 @@ private void OnRemove(EntityUid uid, LightningComponent component, ComponentRemo
     /// <param name="user">Where the lightning fires from</param>
     /// <param name="target">Where the lightning fires to</param>
     /// <param name="lightningPrototype">The prototype for the lightning to be created</param>
-    public void ShootLightning(EntityUid user, EntityUid target, string lightningPrototype = "Lightning")
+    /// <param name="triggerLightningEvents">if the lightnings being fired should trigger lightning events.</param>
+    public void ShootLightning(EntityUid user, EntityUid target, string lightningPrototype = "Lightning",  bool triggerLightningEvents = false)
```

Comment:
The old behavior was default true. Please adjust the default to preserve the old behavior. Otherwise, this is a breaking API change and needs to be in a separate PR.